### ISSUE TITLE
Sequeler crashes when trying to display tables that contain a json field

### ISSUE THREAD

**Issue by stefpankov** [Alecaddd/sequeler#86]
This is the error message I'm getting after running a query for a json column. Upon clicking ok the app crashes.
When browsing from the sidebar and clicking on the content button the app crashes without an error message.

![screenshot from 2018-02-21 12-08-02](https://user-images.githubusercontent.com/12098874/36477098-498d8fb6-1700-11e8-8b57-7b3b20560808.png)

**Comment by Alecaddd** [comment]
This is an issue of the `LibGDA` of Gnome. This library is just awful >_>
Could you share a dump of that table? Maybe just a couple of rows with the data so I can import them into my local DB and test it.

Cheers

**Comment by stefpankov** [comment]
Sorry, I cannot from that particular table but I will create another database with similarities and dump it.
I was wondering if the json columns get returned as strings from `LibGDA` or if they're being cast to something else. This might be why it crashes if it's trying to display a string but the cell is given some other data it can't properly stringify, just some thoughts, I'll get back to you with a dump.

**Comment by Alecaddd** [comment]
Indeed, this is an issue of the `LibGDA`. That library is really strict in terms of handling data types. It crashes even if the FIELD is set as `INT` but the data is a `INT64`. It's a really outdated library that saw the last official release in 2015.
I'm more and more convinced that I should drop it completely.

**Comment by stefpankov** [comment]
Here is a very simple database dump that reproduces the issue. Try first opening the content of the users table, seeing the one record and then opening the content of the vehicles table that causes the app to crash. 

```
SET NAMES utf8;
SET time_zone = '+00:00';
SET foreign_key_checks = 0;
SET sql_mode = 'NO_AUTO_VALUE_ON_ZERO';

SET NAMES utf8mb4;

DROP TABLE IF EXISTS `users`;
CREATE TABLE `users` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `username` varchar(32) COLLATE utf8mb4_unicode_ci NOT NULL,
  `email` varchar(64) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;

INSERT INTO `users` (`id`, `username`, `email`) VALUES
(1,	'bwayne',	'bruce@waynemanor.com');

DROP TABLE IF EXISTS `vehicles`;
CREATE TABLE `vehicles` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  `colors` json NOT NULL,
  `user_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  KEY `user_id` (`user_id`),
  CONSTRAINT `vehicles_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;

INSERT INTO `vehicles` (`id`, `name`, `colors`, `user_id`) VALUES
(1,	'Batmobile',	'[{\"type\": \"hex\", \"value\": \"000000\"}, {\"type\": \"rgb\", \"value\": \"56,62,66\"}]',	1);
```

**Comment by jtojnar** [comment]
LibGDA development is active again (>500 commits in last 90 days), maybe report it in their [issue tracker](https://gitlab.gnome.org/GNOME/libgda/issues).

**Comment by Alecaddd** [comment]
@stefpankov hey, would you be able to test this again? I tested a pretty big json table and it worked without problems.
Let me know
